app/forms/apo_form.rb

```diff
@@ -79,6 +79,19 @@ def collection_radio
     'none'
   end
 
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
+  def default_object_rights
+    return default_rights if model.nil?
+    return "loc:#{model.administrative.defaultAccess.readLocation}" if default_rights == 'location-based'
+    return 'cdl-stanford-nd' if model.administrative&.defaultAccess&.controlledDigitalLending
```

Comment:
why do you need &. in this line but not on the prior line?

Reply:
Thanks for the catch; updated in latest commit.